open-reac/src/main/java/com/powsybl/openreac/Reports.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openreac;
+
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+
+/**
+ * @author Joris Mancini <joris.mancini_externe at rte-france.com>
+ */
+public final class Reports {
+
+    private Reports() {
+        // Should not be instantiated
+    }
+
+    public static Reporter createVoltageInitReporter(Reporter reporter, String networkId) {
```

Comment:
Please remove any reference to voltage init or voltage initialisation : this is linked to our legacy. Here it is called OpenReac. Maybe adding a reference to the objective function is needed to fit totally with our legacy.

Reply:
Sure, will do !